docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,71 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+You can set up a local agent to execute pipelines launched in Dagster+. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
+
+:::warning
+Local agents aren't well suited for most production use cases. This is because local agents:
+- Don't have the same CI / CD update strategies as the other agents.
+- Need a separate process running on the sever hosting the agent to pull in changes to your Dagster project.
+- Need to be restarted to deploy changes.
+:::
+
+:::note
+If you're running the local agent in production, make sure you've set up a supervisor to automatically restart the agent process if it crashes. You'll also want a system in place to alert you if the VM or container dies, or to automatically restart it.
+:::
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- **Organization Admin** permissions in your Dagster+ account.
+- **To install the `dagster-cloud` CLI** in the same environment where the agent will run. We recommend using a Python virtual environment for your Dagster application code and its dependencies.
+    ```bash
+    pip install dagster-cloud
+    ```
+
+</details>
+
+## Step 1: Generate an agent token
+
+Your local agent will need a token to authenticate with your Dagster+ account. To generate an agent token:
```

Comment:
Ideally this section will link to the token guide, to make sure we're not duplicating content. Let's leave this for now and I'll move it in a follow up